src/templates/categories-status-index.macro.html

```
@@ -62,26 +62,28 @@ <h5 class='modal-title' id='createCatStatModalLabel'><i class='fas fa-fw fa-plus
           <span aria-hidden='true'>&times;</span>
         </button>
       </div>
-      <div class='modal-body'>
-          <div class='d-flex justify-content-between mb-2'>
-            <label for='createCatStatName' class='col-form-label'>{{ 'Name'|trans }}</label>
-            <input class='form-control col-md-6' type='text' name='name' placeholder='{{ newNamePlaceholder }}' required id='createCatStatName' />
-          </div>
-          <hr>
+      <form id='createCatStatForm' aria-label='{{ 'Edit event'|trans }}'>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)